jsapp/scss/components/_kobo.modal.scss

```diff
@@ -746,6 +746,94 @@ $z-modal-x: 10;
   }
 }
 
+.form-media__label {
+  font-weight: 400;
+  font-size: 16px;
+  text-align: center;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 5px;
+}
+
+.form-media__url-input {
+  width: 75%;
+  border: 1px solid #80808063;
```

Comment:
Please define colors in `_mdl.scss` file. Or preferably use existing ones :) And rgba would be easier to understand the alpha part.